src/generators/emoji/generator.ts

```diff
@@ -118,7 +119,7 @@ function replaceNumbers(str: string) {
 
 function removePunctuations(str: string): EmojiString {
   const lowerStr = str.toLowerCase();
-  if (!/[,|.|!|?|;|(|)|"]/g.test(str)) {
+  if (!/[,|.|!|?|;|(|)|"|'|:]/g.test(str)) {
     return {
```

Comment:
bitte unleserlicher

Reply:
Ne ernsthaft, könntest du das irgendwie auslagern, in eine Methode oder so um zu beschreiben was dieses RegEx macht? RegEx schreiben ist einfacher als bestehende zu lesen.